h/static/scripts/directives/thread.coffee

```diff
@@ -132,6 +132,7 @@ ThreadController = [
     # thread filter attached to this thread, it will return true.
     ###
     this.matchesFilter = ->
+      return false if @container?.message?.$orphan # We don't want orphans
```

Comment:
If you remove this, the root will semi-appear in search results. (It will
say that there is one match, and then show nothing.)
2015.03.24. 19:25 ezt írta ("Randall Leeds" <notifications@github.com>):

> In h/static/scripts/directives/thread.coffee
> <https://github.com/hypothesis/h/pull/2080#discussion_r27058142>:
>
> > @@ -132,6 +132,7 @@ ThreadController = [
> >      # thread filter attached to this thread, it will return true.
> >      ###
> >      this.matchesFilter = ->
> > +      return false if @container?.message?.$orphan # We don't want orphans
>
> Is this necessary? The conversation roots are hidden by the
> shouldShowThread, which makes sense to me because it's view specific
> (don't show orphans *in the sidebar*). For the other view controllers,
> this doesn't apply, so that seems like the only pivot needed. This one can
> go away, perhaps?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/2080/files#r27058142>.
>